designer/client/src/components/graph/node-modal/fragment-input-definition/settings/variants/fields/UserDefinedListInput.tsx

```diff
@@ -166,7 +168,7 @@ export const UserDefinedListInput = ({
 
     return (
         <FormControl>
-            <SettingLabelStyled>{t("fragment.addListItem", "Add list item:")}</SettingLabelStyled>
+            <SettingLabelStyled>{inputLabel || t("fragment.addListItem", "Add list item:")}</SettingLabelStyled>
```

Comment:
Why we still have this fallback? Let's remove it if it is unused or at least change to "List items" to avoid further bugs

Reply:
The intention was to leave this property optional, and not force someone who wants to unit test this component to think about label value, but since we don't have any unit/integration tests (only e2e) i will remove this fallback

Reply:
Done ✔️